README.md

```diff
@@ -69,3 +60,44 @@ public final class ResolveFlags {
   }
 }
 ```
+
+## Metrics emission (EXPERIMENTAL)
+
+Confidence offers APIs to collect metrics from your application in the form of `EVENTS`.
+
+The experimental `Confidence` interface allows to both:
+- Configure the OpenFeature Provider for flag resolves
+- Emit events
+
+Usage example:
+
+```java
+import com.spotify.confidence.ConfidenceValue;
+import dev.openfeature.sdk.FeatureProvider;
+
+public final class ResolveFlags {
+
+  public static final String CLIENT_TOKEN = "<>";
+
+  public static void main(String[] args) {
+    final Confidence confidence = Confidence.builder(CLIENT_TOKEN).build();
+    final FeatureProvider provider = new ConfidenceFeatureProvider(confidence);
+    // Flags are operated via the same OpenFeature Client APIs
+    OpenFeatureAPI.getInstance().setProvider(provider);
+
+    // Additionally, events can be emitted
+    confidence.send("my-event", ConfidenceValue.of("event-value"));
+  }
+}
+```
+### The `send()` API currently supports
+- Setting a custom event's payload with all the Confidence-supported types via the `ConfidenceValue` interfaces
+- Automatically including the OpenFeature's `Evaluation Context` detected at the time `send()` is called
+  - _Note: this only considers the global Evaluation Context set at the OpenFeatureAPI level_
+### Setting a shared `Event Context` that is going to be appended to each event
+
+```java
+    final Confidence confidenceWithContext = confidence.withContext(ConfidenceValue.of("context-value"));
+    confidenceWithContext.send("my-event", ConfidenceValue.of("event-value"));
+```
+The "my-event" event in the example above will contain fields for "event-value", "context-value" and the Evaluation Context data set via `OpenFeatureAPI.getInstance().setEvaluationContext(...)`.
```

Comment:
nit: 
```suggestion
The "my-event" event in the example above will contain fields for "event-value", "context-value" and the Evaluation Context data set via `OpenFeatureAPI.getInstance().setEvaluationContext(...)`.

We suggest you you this `withContext()`-API to send events which are "request context" based events as well.

```

Reply:
> We suggest you you this `withContext()`-API to send events which are "request context" based events as well.

Can you re-phrase slightly or explain here what you mean?